chia/farmer/farmer_api.py

```diff
@@ -249,6 +249,10 @@ async def new_signage_point(self, new_signage_point: farmer_protocol.NewSignageP
         await self.farmer.server.send_to_all([msg], NodeType.HARVESTER)
         if new_signage_point.challenge_chain_sp not in self.farmer.sps:
             self.farmer.sps[new_signage_point.challenge_chain_sp] = []
+        if new_signage_point in self.farmer.sps[new_signage_point.challenge_chain_sp]:
+            self.farmer.log.debug(f"Duplicate signage point {new_signage_point.signage_point_index}")
+            return
```

Comment:
I see you want to reintroduce the plain returns :^)

Reply:
oops!